src/Bicep.LangServer/Handlers/BicepDeploymentParametersHandler.cs

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
When writing parameters into params file, they end up in reverse order (I know this is because you're using jsonEditor, but you can at least insert them in the proper order relative to each other by writing them in reverse order).  Not perfect but better.

Reply:
Updated here:
https://github.com/Azure/bicep/commit/25fc09c8d76f44ea4123b6c9f6defcc88b2bf23c